edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/Constants.cs

```diff
@@ -86,5 +94,16 @@ public static class EdgeDeployment
         public const string HostIPC = "host";
         public const string HostNetwork = "host";
         public const string HostNetworkDnsPolicy = "ClusterFirstWithHostNet";
+
+        public const string ResourceMemoryLimitKey = "memory";
+
+        public const string ResourceCpuLimitKey = "cpu";
+
+        public const string ProxyMemoryLimitDefault = "25M";
+
+        public const string ProxyCpuLimitDefault = "5m";
+        public const string AgentMemoryLimitDefault = "200M";
+
+        public const string AgentCpuLimitDefault = "50m";
```

Comment:
AKS Monitoring insights.  I have been running edge runtimes on an AKS cluster and tracking CPU/memory max and increased it by a comfortable margin.

Reply:
I see. I would really consider these to be requested values not limit values and give a fairly enough room for the limit. This is particularly for the memory limits because it's stated in docs [1] that:_"If a Container exceeds its memory limit, it might be terminated. If it is restartable, the kubelet will restart it, as with any other type of runtime failure."_

[1] https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

Reply:
Good point about the termination. Let's confirm with ASE on the requirement to make the final call. As per ASE, they wanted to set a limit and use K8s Qos class to ensure it.

Reply:
Kept default resource settings, but the user may fine tune these settings by changing the "resource" section in the Helm charts.  (They may also set `resources: null` and that will remove them from the charts and configuration.)